Automattic/studio
# Add import tab UI for existing sites

- Closes https://github.com/Automattic/dotcom-forge/issues/8157

## Proposed Changes

- Add UI to import by drag and drop a file.
- Add logic to pick a file from file system and import a site.
- Add tests to cover drag and drop hook and new import export tab.

## Testing Instructions



- Run `STUDIO_IMPORT_EXPORT=true npm start`
- Access an existing site
- Select Import / Export tab
- Click on the Import Drop zone and Select a valid file to import
- Observe the import works as expected displaying a confirmation modal before starting the import.
- Click on Start again
- Drag and drop a valid file to import
- Observe the import works as expected
- Do not click on Start again and Drag and drop a valid file to import
- Observe the import works as expected
- Repeat the steps with any other file that is not valid
- Observe a message of error


https://github.com/user-attachments/assets/ac07e062-d69c-448c-ae36-6cfdebb213cf


## Pre-merge Checklist
